Gemfile

```diff
@@ -54,14 +53,20 @@ gem 'jquery-ui-rails'
 # CORS
 gem 'rack-cors', :require => 'rack/cors'
 
-gem 'rainbow'
 
 # Use ActiveModel has_secure_password
 gem 'ruby-saml'
 
 gem 'strong_migrations'
 gem 'will_paginate'
 
+group :production do
```

Comment:
These gems should likely be available in environments other than production

Reply:
They are pulled in via `rubocop` when it is installed